deployment/gke.sh

```diff
@@ -51,11 +51,16 @@ else
 fi
 
 # Set BQ dataset
-if [ "$PROJECT_ID" == "datcom-mixer-staging" ]; then
-  perl -i -pe's/BQ_DATASET/google.com:datcom-store-dev.dc_v3_staging_clustered/g' deployment.yaml
-else 
-  perl -i -pe's/BQ_DATASET/google.com:datcom-store-dev.dc_v3_clustered/g' deployment.yaml
+if [ "$PROJECT_ID" == "datcom-mixer" ]; then
+  bq_dataset_input_file="./prod_bq_dataset.txt"
+else
+  bq_dataset_input_file="./staging_bq_dataset.txt"
 fi
+while read -r line; do
+  BQ_DATASET=$line
+done < "$bq_dataset_input_file"
+export BQ_DATASET
+perl -i -pe's/BQ_DATASET/$ENV{BQ_DATASET}/g' deployment.yaml
```

Comment:
I'm curious why perl was used, long time no see :)

this is fine for consistency, but an alternative would be to do:

    sed -i "s/BQ_DATASET/$BQ_DATASET/g" deployment.yaml

which doesn't need "export BQ_DATASET".

Reply:
I got sed: 1: "deployment.yaml": extra characters at the end of d command
Maybe if we use sed, different OS will respond different, vs perl.